### ISSUE TITLE
not working for me

### ISSUE THREAD

**Issue by ericjamesonline** [4ndv/HexaGone#1]
i installed the poo script with tampermonkey but the profile pic is still a hex
i'm using safari btw

<img width="600" alt="Screen Shot 2022-01-22 at 1 09 28 AM" src="https://user-images.githubusercontent.com/86898409/150627258-80b071a3-2c3c-45c0-b98a-6cb64ef3a357.png">

**Comment by 4ndv** [comment]
Hi! Please open devtools console and send result of this:

```javascript
document.querySelector("#hex-hw-shapeclip-clipconfig")
```

**Comment by ericjamesonline** [comment]
couldn't find any results in safari devtools or tampermonkey code

**Comment by 4ndv** [comment]
Please open javascript console in devtools, paste the code and send a screenshot of the result

**Comment by 4ndv** [comment]
Sadly, I'm unable to check in tempermonkey on safari myself, because apparently there is no free way to install tempermonkey for safari

**Comment by ericjamesonline** [comment]
nothin
<img width="1440" alt="Screen Shot 2022-01-25 at 5 07 23 PM" src="https://user-images.githubusercontent.com/86898409/151067832-dea05795-c1ea-4c0c-8d40-6c2cc7943ee4.png">

the JS console in safari is a list of logs anyway

**Comment by 4ndv** [comment]
You need to copy the code in my previous reply, paste it in the bottom (click after ">" icon) and press return

**Comment by ericjamesonline** [comment]
ok here's the result

```
<clippath clipPathUnits="objectBoundingBox" id="hex-hw-shapeclip-clipconfig" transform="scale(0.005 0.005319148936170213)">…</clippath> = $1
```

**Comment by 4ndv** [comment]
Thanks! Will try to test it in a couple of hours in safari with tampermonkey

**Comment by 4ndv** [comment]
I've looked into it, looks like a Safari bug to me:

https://github.com/Tampermonkey/tampermonkey/issues/296

TLDR: Safari enforces Content Security Policy when it shouldn't, which breaks userscripts on pages with strict CSP setups like twitter. Sadly, but there is no something I can fix about it
